app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.find(params(:id))
```

Comment:
I don't think `users#new` is ever  used.  If it is.. this would give a syntax error.